spec/helpers/handle_helper_spec.rb

```diff
@@ -1,21 +1,20 @@
 require "spec_helper"
 include HandleHelper
 
-describe HandleHelper, unless: $in_travis do
+describe HandleHelper do
   let(:bills_file) { FactoryGirl.create(:bills_complete_file) }
-  `mysql -u "#{ENV["HANDLE_TEST_USERNAME"]}" < "#{Rails.root}"/spec/fixtures/files/handlesTEST.sql`
```

Comment:
Then I'm still faced with the same issue of getting the test values into the jobs unless i change the jobs to accept the client like handle_helper does